.github/workflows/ci-cd.yaml

```diff
@@ -55,8 +54,31 @@ jobs:
         uses: helm/kind-action@v1.10.0
         with:
           version: v0.24.0
-      - name: Run tests
-        run: ./test.sh ${{ matrix.args }}
+      - name: Run trino tests
```

Comment:
Keep this step name as `Run tests`, but change the job name on line 33 to `trino-test`, so it is distinguishable from gateway tests.